source/operations/network-encryption.rst

```diff
@@ -30,8 +30,9 @@ Enabling TLS
    The TLS certificate generation process is as follows:
 
    - The Operator generates a Certificate Signing Request (CSR) associated to the Tenant.
-     The :abbr:`CSR (Certificate Signing Request)` includes the appropriate DNS SubjectAlternateNames (SAN) for the Tenant services and pods.
-     
+     The :abbr:`CSR (Certificate Signing Request)` includes the appropriate DNS Subject Alternate Names (SANs) for the Tenant services and pods.
+     If not using a wildcard SAN, the TLS certificate SAN **must** apply to the hostname for its parent node.
```

Comment:
I'm unclear if this requirement is relevant for k8s, since Operator manages many things.

Also I don't know much about DNS SANs, maybe the wording can be improved.

Reply:
Hm - chances are this is not necessary. *That said* we do support users who specify custom certificates.

I would check with the K8s team what the default cert config is and go with that.